apps/explorer/src/pages/transaction-result/programmable-transaction-view/TransactionsCard.tsx

```diff
@@ -24,7 +24,7 @@ export function TransactionsCard({ transactions }: TransactionsCardProps) {
 		const [[type, data]] = Object.entries(transaction);
 
 		return (
-			<TransactionBlockCardSection key={index} title={type} defaultOpen={defaultOpen}>
+			<TransactionBlockCardSection defaultOpen key={index} title={type}>
```

Comment:
the `defaultOpen` variable can probably just get inlined down below now

Reply:
done